### PR TITLE
[Fix]fix lazy open commit info lose

### PR DESCRIPTION
**Pull request by sollhui**
## Proposed changes

**problem**
when open enable_lazy_open_partition config, there is a possibility that the open partition rpc will  perform the open operation after the txn commit, which will cause the commit to fail and the load fail.

**What is this pr task**
- Therefore, to solve this problem,When sending the last RPC to obtain the commit information, it is necessary to wait for all lazy opens to end.
- a more comprehensive case has been added for lazy open testing.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by HHoflittlefish777** [comment]
run buildall

File: regression-test/suites/rollup_p0/test_materialized_view_lazy_open.groovy [hunk 1]
```diff
@@ -70,6 +75,58 @@ suite("test_materialized_view_lazy_open", "rollup") {
...
+            PARTITION BY RANGE(k1)
+            (
+               PARTITION p1 VALUES LESS THAN ("2000-01-01"),
+               PARTITION p2 VALUES LESS THAN ("2010-01-01"),
+               PARTITION p3 VALUES LESS THAN ("2020-01-01")
+            )
+            DISTRIBUTED BY HASH(k1) BUCKETS 32 
+
+            properties(
+                "light_schema_change" = "true",
```
Comment (freemandealer) [inline regression-test/suites/rollup_p0/test_materialized_view_lazy_open.groovy:101]: Even though we enable schema change here, we have not triggered schema change yet.

File: be/src/vec/sink/vtablet_sink.cpp [hunk 2]
```diff
@@ -320,6 +320,9 @@ Status VNodeChannel::init(RuntimeState* state) {
 
     _rpc_timeout_ms = state->execution_timeout() * 1000;
     _timeout_watch.start();
+    if (config::enable_lazy_open_partition) {
+        _lazy_open_timeout_watch.start();
+    }
```
Comment (freemandealer) [inline be/src/vec/sink/vtablet_sink.cpp:325]: no need to start stop watch here.

File: be/src/vec/sink/vtablet_sink.cpp [hunk 3]
```diff
@@ -516,7 +519,7 @@ Status VNodeChannel::open_wait() {
 }
 
 void VNodeChannel::open_partition(int64_t partition_id) {
-    _timeout_watch.reset();
+    _lazy_open_timeout_watch.reset();
```
Comment (freemandealer) [inline be/src/vec/sink/vtablet_sink.cpp:522]: use stop watch as local var and start here.

File: be/src/vec/sink/vtablet_sink.h [hunk 4]
```diff
@@ -302,6 +304,7 @@ class VNodeChannel {
     int _rpc_timeout_ms = 60000;
     int64_t _next_packet_seq = 0;
     MonotonicStopWatch _timeout_watch;
+    MonotonicStopWatch _lazy_open_timeout_watch;
```
Comment (freemandealer) [inline be/src/vec/sink/vtablet_sink.h:307]: use local var instead of member?

**Comment by HHoflittlefish777** [comment]
run buildall

**Review by freemandealer — APPROVED** [review]
LGTM

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**